lib/Elevate/Components/MySQL.pm

```diff
@@ -360,6 +363,51 @@ sub _blocker_mysql_upgrade_in_progress ($self) {
     return 0;
 }
 
+sub _blocker_mysql_database_corrupted ($self) {
+
+    # Do not perform this check for remote mysql
+    return 0 unless Cpanel::MysqlUtils::MyCnf::Basic::is_local_mysql();
+
+    # We cannot run mysqlcheck if mysql is not running
+    if ( !Cpanel::MysqlUtils::Running::is_mysql_running() ) {
+
+        # No blocker if it is not running because it is not enabled
+        # No need to check the database integrity if MySQL is disabled
+        return 0 unless Cpanel::Services::Enabled::is_enabled('mysql');
+
+        my $output = $self->ssystem_capture_output(qw{/scripts/restartsrv_mysql});
+
+        WARN('Database server was down, starting it to check database integrity');
+        unless ( Cpanel::MysqlUtils::Running::is_mysql_running()
+            && grep { /mysql (?:re)?started successfully/ } @{ $output->{stdout} } ) {
+
+            return $self->has_blocker( <<~"EOS" );
+            We have not been able to successfully start the database server
+            to check the integrity of the databases.
```

Comment:
FIX: Lets improve this message to be more helpful to the customer.  Maybe something like:

```
Unable to to start the database server to check database integrity.  Additional information can be found in the error log located at ...

To attempt to start the database server, execute ...
```